src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2846,6 +2846,14 @@ private bool TryGetReturnType(out TypeWithAnnotations type, out FlowAnalysisAnno
         public override BoundNode? VisitLocal(BoundLocal node)
         {
             var local = node.LocalSymbol;
+
+            // Avoid cycles in nullable analysis.
+            if ((local as SourceLocalSymbol)?.IsVar == true && local.ForbiddenZone?.Contains(node.Syntax) == true)
```

Comment:
This is just a stylistic suggestion.

```suggestion
            if (local is SourceLocalSymbol { IsVar: true } && local.ForbiddenZone?.Contains(node.Syntax) == true)
```

Reply:
Thanks.